co-log-core/src/Colog/Core/IO.hs

```diff
@@ -0,0 +1,37 @@
+module Colog.Core.IO 
+    ( logStringStdout
+    , logStringStderr
+    , logStringHandle
+    , withLogStringFile
+    ) where
+
+import Control.Monad.IO.Class (MonadIO, liftIO)
+import System.IO (Handle, IOMode( AppendMode ), hPutStrLn, stderr, stdout, withFile)
+import Colog.Core.Action (LogAction (..))
+
+
+----------------------------------------------------------------------------
+-- String
+----------------------------------------------------------------------------
```

Comment:
I think that this comment is redundant, since this module has only `String` section